docs/calendar-review.md

```diff
@@ -0,0 +1,214 @@
+<code>  Temporal.PlainMonthDay.from( \
+    { monthCode: '05L', day: 12, calendar: 'hebrew' } \
+  ); \
+</code>
+
+It's not clear how we'd model holidays if the calendar isn't stored in the instance.
+*   Code that wants to return a date along with its calendar can no longer return a single Temporal instance. Instead, it must return a compound object or serialized string. For example, a date picker that returned a date in the user's preferred calendar could no longer return a <code>Temporal.PlainDate</code>. Instead, it'd have to return a tuple or compound object like <code>{date, calendar}</code>.  Passing around compound objects (or extra calendar parameters) would complicate all calendar-using Temporal code.
+
+The Temporal champions believe that the ergonomic challenges of this alternative are worse than the work required to mitigate the Unexpected Calendar Problem. 
+
+### Alternative 2: Separate ISO and Calendar Properties
+
+It would be possible to retain the calendar in Temporal objects' data model without being vulnerable to the Unexpected Calendar Problem, but doing so would involve a significant increase in Temporal surface area. Below is one way it could be done. 
+
+*   Offer two sets of fields, e.g. `.isoMonth` vs. `.calendarMonth` (or `month` vs. `calendarMonth` for brevity and to encourage developers who are unfamiliar with calendars to be nudged to use ISO fields?)
+    *   The number and types of `calendar*` fields are completely up to the calendar implementation.  Built-in calendars will all offer `calendarDay`, `calendarMonth`, `calendarMonthCode`, `calendarYear`, and sometimes `calendarEra` and `calendarEraYear` too.
+    *   Temporal APIs that accept property bag parameters will accept either calendar fields or ISO fields, BUT NOT BOTH. `date.with({calendarYear: 5780, isoMonth: 12}) `should throw. BTW, this may be enforceable via TS and ESLint for a nice design-time DX.
+    *   Trying to read `calendar*` fields would throw if the calendar is ISO because there is no human calendar defined.
+    *   Non-field convenience properties (e.g. `dayOfYear`, `inLeapYear`, etc.) would also be duplicated. The non-`calendar` variants of these fields would report the ISO values.
+*   `Temporal.Duration` would gain a `calendar` field.  
+    *   Instances using the ISO calendar would only support ISO fields: `days`, `weeks`, `months`, and `years`. 
+    *   Instances using a non-ISO calendar would only support calendar fields: `calendarDays`, `calendarWeeks`, `calendarMonths`, and `calendarYears`. 
+    *   String parsing and serialization would use calendar annotations like other Temporal types, e.g. `'P2D[u-ca-chinese]'`. Strings without an annotation would assume the ISO calendar.
+    *   Question: should `date.add({calendarMonths: 1})` use `date.calendar`? Or should `date.add({calendarMonths: 1, calendar: 'chinese'})` be required even if `date.calendar==='chinese'`?
```

Comment:
This section kind of mixes up the modifications that would need to be made in order to take this approach, with the consequences of making those modifications. FWIW, I think the document is fine this way, so no need to block merging on this, but it does read slightly confusingly.

Reply:
I removed it.  I think there are enough other bullets in this section.  (OK @justingrant?)